### DISCUSSION TITLE
A large number of IPC timed out. method errors

### DISCUSSION THREAD

**Discussion by db7689206** [colyseus/colyseus#710] [Q&A]
I used colyseus on the back end of my game and got crazy when there were too many users: IPC timed out. method: undefined, args: []。
Ask for my help,Thanks.

**Comment by endel** [comment]
Hi @db7689206, this error generally means the Redis instance is taking too long to respond. Are you possibly excessively using the presence API?

**Reply by db7689206** [reply]
Hi, but my redis isn't under any pressure.But I opened multiple server instances.I've improved the responsiveness of my server, but I've also gotten more errors. Not only this Error, but also error: the seat reservation expired is also very many.

**Comment by db7689206** [comment]
![image](https://github.com/colyseus/colyseus/assets/21011863/fb7ac73e-3784-4ff3-9d5d-47c79e3c1d0f)
Large number of errors： seat reservation expired

**Comment by endel** [comment]
Hi @db7689206, this is very likely a misconfiguration in the deployment. I'd suggest giving a read to this thread: https://github.com/orgs/colyseus/discussions/578#discussioncomment-6174564

If that doesn't help you, please share how your deployment config is

**Reply by db7689206** [reply]
I read through it and it didn't solve my problem, my architecture is that the ingress configuration forwards, hashes the payload to my pod based on the $remote, and I have multiple Pods in one deployment.
![image](https://github.com/colyseus/colyseus/assets/21011863/74978812-a46f-4bc3-a1a2-f3a203896942)
![image](https://github.com/colyseus/colyseus/assets/21011863/ad572f60-4048-4477-87c2-22d88cc59ae0)
Add below:
These errors occur only when there are too many users, but there don't seem to be too many users. Only a few hundred clients
![image](https://github.com/colyseus/colyseus/assets/21011863/687f93fc-0a8e-476a-8715-53464b3ddbaf)

**Comment by db7689206** [comment]
Sometimes there is an error when linking websocket. I checked the request and found that the authentication header is not there, but this is not what I set in the code.
normal:
![img_v3_029b_aa0b758c-c172-45cb-9661-9a422c2ad29h](https://github.com/colyseus/colyseus/assets/21011863/6ac0ed72-c933-4c30-8fc3-4d64fb49c4a3)
error:
![img_v3_029b_2e911ad2-726f-4d2b-a430-d542defdf01h](https://github.com/colyseus/colyseus/assets/21011863/ce2adc37-2e9e-4e62-adc9-7ff22d45264c)
